packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -127,7 +127,7 @@ export class Image extends BaseComponent<IImageProps, IImageState> {
   private _evaluateImage(): boolean {
     let { src } = this.props;
     let { loadState } = this.state;
-    let isLoaded = (src && this._imageElement.naturalWidth > 0 && this._imageElement.naturalHeight > 0);
+    let isLoaded = (src && this._imageElement.complete);
```

Comment:
It's actually 0. I don't think we can. Can we just ask users not to use the Image component for SVGs? I was reading about them and they seem very different than images.